src/components/Accordion/Accordion.tsx

```diff
@@ -10,14 +10,15 @@ import {
 import Icon from "../Icons/Icon";
 import { IconNames, IconSizes } from "../Icons/IconTypes";
 import generateUUID from "../../helpers/generateUUID";
-
+import { AccordionTypes } from "./AccordionTypes";
 export interface AccordionContentDataProps {
+  accordionType?: AccordionTypes;
```

Comment:
Instead of making this required, can this be optional with a default value of `AccordionTypes.Default`?

Reply:
Sorry, I mixed up my syntax.  Yes, I now see that prop is optional.  And now that I look even closer at the code, I see am seeing the logic that is defaulting to `default`.